task_7/dima.greshchook/Factorial/src/test/java/WrapperForFactorialTest.java

```diff
@@ -0,0 +1,26 @@
+import static org.junit.Assert.*;
+
+public class WrapperForFactorialTest {
+
+  @org.junit.Test
```

Comment:
Use import instead of adding the full name with package.
It should be just:
**@Test**